### PR TITLE
rstudio-server v0.99.484

### PR DESCRIPTION
**Pull request by randy3k**
Update to rstudio-server v0.99.484

### CODE REVIEW

File: rstudio-server.rb [hunk 1]
```diff
@@ -1,13 +1,7 @@
 class RstudioServer < Formula
   homepage "http://www.rstudio.com"
-  url "https://github.com/rstudio/rstudio/archive/v0.99.473.tar.gz"
-  sha256 "a402995251393d50fa70e2e02188a8b32dc34e067f2f8461f087bf71f643585c"
-
-  bottle do
-    sha256 "ce5defab059bdc9452c007e94772d957594688bed899061b4fe6f10864ec2c3e" => :yosemite
-    sha256 "d6da8dd62a9e94705b14890bbc6932f9abe4ac745578d368c54cc2d783636819" => :mavericks
-    sha256 "fe343ba1961d3ba11ab86597084e180de97edea88cf1d2c4470b269a3225b2ff" => :mountain_lion
-  end
```
Comment (dpo) [inline rstudio-server.rb]: Please don't alter the `bottle` section. It will be updated automatically.

Comment (randy3k) [inline rstudio-server.rb]: I've revert the `bottle` section and rebased the code.

**Comment by smartinsightsfromdata** [comment]
@randy3k do you plan to update this?

**Comment by randy3k** [comment]
@smartinsightsfromdata see #3301.

**Pull request closed without merging**